datastore/longhorn.go

```diff
@@ -313,6 +313,16 @@ func (s *DataStore) ValidateSetting(name, value string) (err error) {
 		for _, checkKey := range checkKeyList {
 			if value, ok := secret.Data[checkKey]; ok {
 				if strings.TrimSpace(string(value)) != string(value) {
+					switch {
+					case strings.TrimLeft(string(value), " ") != string(value):
+						return fmt.Errorf("there is leading white space in %s", checkKey)
```

Comment:
```suggestion
						return fmt.Errorf("invalid leading white space in %s", checkKey)
```
Could you also fix the others? Thank you.

Reply:
Sure. Modified. Thanks